SAMtags.tex

```diff
@@ -131,6 +132,13 @@ \subsection{Additional Template and Mapping data}
 \item[CC:Z:\tagvalue{rname}]
 Reference name of the next hit; `{\tt =}' for the same chromosome.
 
+\item[CG:B:I,\tagvalue{encodedCigar}]
+Real CIGAR in its binary form if it contains $>$65535 operations. This is
```

Comment:
If this is in binary form, we need to be more specific about which binary format and what parts of the binary format....would it be easier to just store it as a string? What would happen is an "old tool" took the bam and converted it to a sam? how would the tag be encoded?

Reply:
I have changed the wording a little bit, emphasizing that `CG` is encoded exactly the same way as the cigar field in BAM. This allows you to move the entire cigar around with memory copy/move, much more efficient and easier to implement than keeping text CIGAR in the CG tag.

Reply:
if-> if (and only if)